From ba41031694087063d22c2ad6468c4eb996f31c36 Mon Sep 17 00:00:00 2001
From: Ahmad Abbas <ahmada@marvell.com>
Date: Wed, 2 Mar 2016 12:29:15 +0200
Subject: [PATCH 0918/1240] menuconfig: mvebu: menuconfig clean up - serial
 driver

gathered all menuconfig serial driver options under one menu.

Change-Id: I46b12ca4c4178dc1afe0fd799ee755234db9c385
Signed-off-by: Ahmad Abbas <ahmada@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/27983
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: Hanna Hawa <hannah@marvell.com>
---
 drivers/serial/Kconfig | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/serial/Kconfig b/drivers/serial/Kconfig
index 3bbf117..3fc25fb 100644
--- a/drivers/serial/Kconfig
+++ b/drivers/serial/Kconfig
@@ -1,3 +1,5 @@
+menu "Serial Driver"
+
 config DM_SERIAL
 	bool "Enable Driver Model for serial drivers"
 	depends on DM
@@ -27,3 +29,5 @@ config MVEBU_UART_ARLP
 	  for UART driver on ArmadaLP, which
 	  is not using COM1 NS16550 driver.
 	  And base address is configured in FDT file.
+
+endmenu
-- 
1.9.1

